ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Bug Report
+about: Create a report to help us improve.
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+<!-- A clear and concise description of what the bug is.-->
+
+**To Reproduce**
+Steps to reproduce the behavior:
```

Comment:
Interchange has this thing about minimal reproducible example that I kinda like: https://github.com/openforcefield/openff-interchange/blob/main/.github/ISSUE_TEMPLATE/BUG_REPORT.md?plain=1#L13